Index: Week1/004.Rainbow/src/Rainbow.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import java.awt.*;\r\nimport java.awt.geom.*;\r\nimport javafx.application.Application;\r\nimport static javafx.application.Application.launch;\r\nimport javafx.scene.Group;\r\nimport javafx.scene.Scene;\r\nimport javafx.scene.canvas.Canvas;\r\nimport javafx.stage.Stage;\r\nimport org.jfree.fx.FXGraphics2D;\r\n\r\npublic class Rainbow extends Application {\r\n    @Override\r\n    public void start(Stage primaryStage) throws Exception {\r\n        Canvas canvas = new Canvas(1920, 1080);\r\n        draw(new FXGraphics2D(canvas.getGraphicsContext2D()));\r\n        primaryStage.setScene(new Scene(new Group(canvas)));\r\n        primaryStage.setTitle(\"Rainbow\");\r\n        primaryStage.show();\r\n    }\r\n    \r\n    \r\n    public void draw(FXGraphics2D graphics) {\r\n\r\n    }\r\n    \r\n    \r\n    \r\n    public static void main(String[] args) {\r\n        launch(Rainbow.class);\r\n    }\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Week1/004.Rainbow/src/Rainbow.java b/Week1/004.Rainbow/src/Rainbow.java
--- a/Week1/004.Rainbow/src/Rainbow.java	(revision f71996447f97b42fa8591dfdb5f5e21c1ed39cc4)
+++ b/Week1/004.Rainbow/src/Rainbow.java	(date 1675089840767)
@@ -20,8 +20,12 @@
     
     
     public void draw(FXGraphics2D graphics) {
+        for(int i = 0; i < 500; i++) {
+            graphics.setColor(Color.getHSBColor(i/500.0f, 1, 1));
+            graphics.drawLine(i, 10, i, 100);
+    }
+    }
 
-    }
     
     
     
